src/wallet/wallet.cpp

```diff
@@ -652,8 +652,12 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFromLoadWallet, CWalletD
                 {
                     int64_t latestNow = wtx.nTimeReceived;
                     int64_t latestEntry = 0;
+                    int64_t blocktime = mapBlockIndex[wtxIn.hashBlock]->GetBlockTime();
```

Comment:
dcousens: It's just a line moved from below so I didn't want to change anything. Arguably it could indeed be cased properly, but I think in this case, the fewer changes the better.